astropy/units/function/core.py

```diff
@@ -50,7 +50,7 @@ class FunctionUnitBase(object):
     function_unit :  `~astropy.units.Unit` or `string`
         By default, the same as the function unit set by the subclass.
     """
-    # vvvvv the following four need to be set by subclasses
```

Comment:
Sorry, I haven't noticed those. Somehow assumed these just remained here as placeholders (I do similar things with easily grep-able strings but with the aim to remove them all). Will revoke these changes.